README.md

```diff
@@ -1,4 +1,6 @@
+
 # prplMesh manifest
+[![Build Status](https://travis-ci.org/prplfoundation/prplMesh-manifest.svg?branch=master)](https://travis-ci.org/prplfoundation/prplMesh-manifest)
```

Comment:
Great!

I think it's worth adding this one on both projects (prplMesh-manifest and prplMesh), but both of them refering to the same travis result.